addons/point_of_sale/static/tests/pos/tours/shipping_date_tour.js

```diff
@@ -0,0 +1,55 @@
+import * as Chrome from "@point_of_sale/../tests/pos/tours/utils/chrome_util";
+import * as Dialog from "@point_of_sale/../tests/generic_helpers/dialog_util";
+import * as ProductScreen from "@point_of_sale/../tests/pos/tours/utils/product_screen_util";
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("ShippingDateTour", {
```

Comment:
Here the name of the tour should be the same as the python test. This way we can find the tour linked to a test more easily. So it should be called `test_pos_order_shipping_date`